assets/src/edit-story/components/helpCenter/navigator/bottomNavigation.js

```diff
@@ -113,7 +117,7 @@ export function BottomNavigation({
           <ArrowWrap isRTL={isRTL}>
             <Icons.ArrowLeft />
           </ArrowWrap>
-          <span>{__('All Tips', 'web-stories')}</span>
+          <span css={secondaryTextStyle}>{__('All Tips', 'web-stories')}</span>
```

Comment:
should this be `style`? I've never seen the css prop before. Does it work similarly?

Reply:
nit for sure but this feels like it might as well be `ArrowText` or something and then just `const ArrowText = styled.span` color: theme.color...` just for continuity.